engine/schema/src/main/java/com/cloud/network/security/dao/SecurityGroupVMMapDaoImpl.java

```diff
@@ -131,7 +131,7 @@ public List<SecurityGroupVMMapVO> listBySecurityGroup(long securityGroupId, Stat
         SearchCriteria<SecurityGroupVMMapVO> sc = ListBySecurityGroupAndStates.create();
         sc.setParameters("securityGroupId", securityGroupId);
         sc.setParameters("states", (Object[])vmStates);
-        return listBy(sc, null, true);
```

Comment:
should we remove this method if it doesn't work?

Reply:
@DaanHoogland 
I think so. but it requires several lines of changes in `GenericDaoBase.java`

Reply:
ok, let's keep it out of scope here